### PR TITLE
Add ut template

### PR DESCRIPTION
**Pull request by shink**
This unit test template contains the following:

- Unit test codes principles and suggestions.
- Two unit test examples: Controller and DAO.

BTW, you can take a look at [the controller unit test demo](https://gitlab.summer-ospp.ac.cn/summer2021/210290131/-/tree/master/Test).

### CODE REVIEW

**Comment by Jave-Chen** [comment]
LGTM @CalvinKirs @wen-hemin

**Review by wen-hemin — COMMENTED** [review]
1.Suggest a test method only covers one case of a method. Method name + Test + "_" + Case Description.
2.It is recommended to add test samples and mock samples for final and static methods.

**Comment by shink** [comment]
@wen-hemin Thank you for your suggestions!!!

**Comment by wen-hemin** [comment]
Is it more appropriate to name the test case like this?
test_methodName_caseDescription
E.g. test_createUser_success

1. Clearer
2. Method name can be copied directly, no need to capitalize the first letter of the method name.

File: blog/zh-cn/ut-template.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+3. **轻量型**
+
+测试应当是秒级甚至是毫秒级的，不应占用过多时间。
+
+> 由于 Spring Boot 启动花费时间较长，因此当待测试对象不依赖 Spring Bean 或 Spring 容器时，应当避免使用 Spring Test 进行单元测试，可以通过直接创建目标类对象的方式实现。
+
+4. **可测性**
+
+可以通过 mock 框架模拟构造方法、静态方法、final 方法等。
+
```
Comment (CalvinKirs) [inline blog/zh-cn/ut-template.md:67]: This statement is not correct. please refer https://dolphinscheduler.apache.org/zh-cn/community/development/unit-test.html

Comment (shink) [reply blog/zh-cn/ut-template.md:67]: Well, dolphinscheduler is using `powermock` which can mock constructor, static and final methods. I will amend it later.

**Comment by shink** [comment]
@wen-hemin Good idea. I'll see if there are other better solutions.

File: blog/zh-cn/ut-template.md (hunk 1 above)
Comment (CalvinKirs) [reply blog/zh-cn/ut-template.md:67]: What I mean is that being able to do and conforming to the norm are two different things. It should not be confused. The test itself is to ensure the quality of the code, not to write tests for testing.

Comment (shink) [reply blog/zh-cn/ut-template.md:67]: Thank you. Got it. I will amend it.

**Pull request merged**